core/src/test/scala/org/apache/spark/serializer/KryoSerializerSuite.scala

```diff
@@ -565,6 +565,20 @@ class KryoSerializerAutoResetDisabledSuite extends SparkFunSuite with SharedSpar
     assert(serInstance.deserialize[Any](serObj) === (obj))
     assert(serInstance.deserialize[Any](byteBuffer) === (obj))
   }
+
+  test("SPARK-27216: kryo serialization with RoaringBitmap") {
+    val bitmap = new RoaringBitmap
+    bitmap.add(1787)
+
+    val safeSer = new KryoSerializer(conf).newInstance()
+    val bitmap2 : RoaringBitmap = safeSer.deserialize(safeSer.serialize(bitmap))
+    assert(bitmap2.equals(bitmap))
+
+    conf.set("spark.kryo.unsafe", "true")
```

Comment:
You are changing the `conf` which also used by other tests within the suite and now the execution order of these tests are important. If the test execution starts with this test and others are executed latter they might break.

Reply:
It can be move to a totally new Suite. I will update it.